docs/docsite/rst/intro_inventory.rst

```diff
@@ -60,6 +60,9 @@ Suppose you have just static IPs and want to set up some aliases that live in yo
 In the above example, trying to ansible against the host alias "jumper" (which may not even be a real hostname) will contact 192.0.2.50 on port 5555.  Note that this is using a feature of the inventory file to define some special variables.  Generally speaking this is not the best
 way to define variables that describe your system policy, but we'll share suggestions on doing this later.  We're just getting started.
 
+.. note:: Values passed in using the ``key=value`` syntax are interpreted as Python literal structure (strings, numbers, tuples, lists, dicts,
```

Comment:
When is 'stringification' performed ? Could you paste an example with unreliable type on definition ?